chassis/core/src/main/java/com/griddynamics/jagger/invoker/v2/SpringBasedHttpClient.java

```diff
@@ -217,4 +228,39 @@ private HttpComponentsClientHttpRequestFactory getRequestFactory(int maxConnPerR
     public Map<String, Object> getClientParams() {
         return newHashMap(clientParams);
     }
+
+    public static class AllowAllCodesResponseErrorHandler implements ResponseErrorHandler {
+        @Override
+        public boolean hasError(ClientHttpResponse response) throws IOException {
+            return false;
+        }
+
+        @Override
+        public void handleError(ClientHttpResponse response) throws IOException {
+            try {
+                response.getStatusCode();
+            } catch (IllegalArgumentException ex) {
+                throw new UnknownHttpStatusCodeException(response.getRawStatusCode(),
+                        response.getStatusText(), response.getHeaders(), getResponseBody(response), getCharset(response));
+            }
+        }
+
+        private byte[] getResponseBody(ClientHttpResponse response) {
+            try {
+                InputStream responseBody = response.getBody();
```

Comment:
Yes, it's ok. 
Look in DefaultResponseErrorHandler - this code is taken from here.